cyber/proto/dag_conf.proto

```diff
@@ -7,11 +7,17 @@ import "cyber/proto/component_conf.proto";
 message ComponentInfo {
   optional string class_name = 1;
   optional ComponentConfig config = 2;
+  optional uint64 adc_index = 3;
+  optional string episode_name = 4;
```

Comment:
Not sure if it's a good idea to add specific simulation support in CyberRT, which is designed to be general and lightweight. Please refer to @unacao @vlin17 .

Reply:
I don't think we need to add any field in dag proto. Everything should be added to task.h.